core/consensus/src/engine.rs

```diff
@@ -47,41 +47,44 @@ impl<Adapter: ConsensusAdapter + 'static> Engine<FixedPill> for ConsensusEngine<
     async fn get_block(
         &self,
         ctx: Context,
-        height: u64,
+        next_height: u64,
```

Comment:
next_height may not be appropriate. For consensus, it gets current_height's block